test/gtest/api_convbiasactiv.cpp

```diff
@@ -168,22 +169,17 @@ TEST_P(ConvBiasActivFwdTest, DriveAPI)
     EXPECT_EQ(status, miopenStatusSuccess);
 }
 
+std::vector<CBATestCase> GetTestValues()
+{
+
+    std::vector<CBATestCase> ret;
+    if(get_handle().GetDeviceName() != "gfx1100")
```

Comment:
```suggestion
    if(miopen::StartsWith(get_handle().GetDeviceName(), "gfx11"))
```

Reply:
And the title of the PR should be changed to "gfx11" or "Navi3x"

Reply:
changed the title as requested.